internal/dbtest/pg_test.go

```diff
@@ -877,3 +887,51 @@ func TestPostgresMultiRange(t *testing.T) {
 	err = db.NewSelect().Model(out).Scan(ctx)
 	require.NoError(t, err)
 }
+
+func TestPostgresJSONB(t *testing.T) {
+	type Item struct {
+		Name string `json:"name"`
+	}
+	type Model struct {
+		ID      int64   `bun:",pk,autoincrement"`
+		Item    Item    `bun:",type:jsonb"`
+		ItemPtr *Item   `bun:",type:jsonb"`
+		Items   []Item  `bun:",type:jsonb,array"`
```

Comment:
What does it mean to use both jsonb and array together?
 I think PostgreSQL will handle the JSONB type, and we only need to consider JSONB when inserting and reading data?

Reply:
Yes.   PostgreSQL doesn't actually support a JSONB array directly, but rather includes several functions (`jsonb_array_elements`) for working with a JSONB arrays on a JSONB column.   Can read more [here](https://hevodata.com/learn/query-jsonb-array-of-objects/).